src/serac/numerics/odes.cpp

```diff
@@ -150,6 +150,7 @@ void SecondOrderODE::Solve(const double time, const double c0, const double c1,
 {
   // assign these values to variables with greater scope,
   // so that the residual operator can see them
+  state_.time  = time;
```

Comment:
We should re-evaluate this interface strategy for ODE solvers. Tracking down the references and ownership of data can get confusing.